hazelcast/src/main/java/com/hazelcast/nio/tcp/WriteHandler.java

```diff
@@ -93,15 +94,32 @@ private void createWriter(String protocol) {
                 outputBuffer.put(stringToBytes(Protocols.CLUSTER));
                 registerOp(SelectionKey.OP_WRITE);
             } else if (Protocols.CLIENT_BINARY.equals(protocol)) {
+                adjustBufferSizesForClientConnection();
```

Comment:
@jerrinot yes, I am saying creating `outputBuffer` is only at here by removing its initialization at constructor